business/openshift_oauth.go

```diff
@@ -0,0 +1,194 @@
+package business
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/kubernetes"
+
+	kube "k8s.io/client-go/kubernetes"
```

Comment:
nitpick - but the Kiali coding standards are to put these third-party imports in the second group and the kiali imports in the third group. So:
```suggestion
	"strings"

	kube "k8s.io/client-go/kubernetes"
	
	"github.com/kiali/kiali/config"
	"github.com/kiali/kiali/kubernetes"
```

Reply:
doesn't look like this was resolved correctly - the current imports are screwed up - double "strings" is causing failures.